deploy/env-uzh-stg/values.yaml

```diff
@@ -0,0 +1,528 @@
+
+  ingress:
+    className: haproxy
+    annotations:
+      cert-manager.io/cluster-issuer: k8s-crt-mgr-iss-stg-apps-default
+    hosts:
+      - host: pwa.klicker.stg.df-app.ch
+        paths:
+          - path: /
+            pathType: ImplementationSpecific
+    tls:
+      - secretName: klicker-frontend-pwa-tls
+        hosts:
+          - pwa.klicker.stg.df-app.ch
+
+  strategy:
+    type: Recreate
+
+  resources:
+    requests:
+      cpu: 50m
+      memory: 50Mi
+    limits:
+      cpu: 200m
+      memory: 200Mi
+
+frontendManage:
+  priorityClassName: staging-workload
+
+  replicaCount: 1
+
+  autoscaling:
+    enabled: false
+
+  image:
+    repository: ghcr.io/uzh-bf/klicker-uzh/frontend-manage-arm
+    pullPolicy: Always
+    tag: clean-argocd
+
+  ingress:
+    className: haproxy
+    annotations:
+      cert-manager.io/cluster-issuer: k8s-crt-mgr-iss-stg-apps-default
+    hosts:
+      - host: manage.klicker.stg.df-app.ch
+        paths:
+          - path: /
+            pathType: ImplementationSpecific
+    tls:
+      - secretName: klicker-frontend-manage-tls
+        hosts:
+          - manage.klicker.stg.df-app.ch
+
+  strategy:
+    type: Recreate
+
+  resources:
+    requests:
+      cpu: 50m
+      memory: 50Mi
+    limits:
+      cpu: 200m
+      memory: 200Mi
+
+frontendControl:
+  priorityClassName: staging-workload
+
+  replicaCount: 1
+
+  image:
+    repository: ghcr.io/uzh-bf/klicker-uzh/frontend-control-arm
+    pullPolicy: Always
+    tag: clean-argocd
+
+  autoscaling:
+    enabled: false
+
+  ingress:
+    className: haproxy
+    annotations:
+      cert-manager.io/cluster-issuer: k8s-crt-mgr-iss-stg-apps-default
+    hosts:
+      - host: control.klicker.stg.df-app.ch
+        paths:
+          - path: /
+            pathType: ImplementationSpecific
+    tls:
+      - secretName: klicker-frontend-control-tls
+        hosts:
+          - control.klicker.stg.df-app.ch
+
+  strategy:
+    type: Recreate
+
+  resources:
+    requests:
+      cpu: 50m
+      memory: 50Mi
+    limits:
+      cpu: 200m
+      memory: 200Mi
+
+backendGraphql:
+  priorityClassName: staging-workload
+
+  replicaCount: 1
+
+  autoscaling:
+    enabled: false
+
+  image:
+    repository: ghcr.io/uzh-bf/klicker-uzh/backend-docker-arm
+    pullPolicy: Always
+    tag: clean-argocd
+
+  appManageSubdomain: manage
+  appStudentSubdomain: pwa
+  appControlSubdomain: control
+
+  debug: true
+  apiDomain: backend-sls.klicker.stg.df-app.ch
+  appManageDomain: manage.klicker.stg.df-app.ch
+  appStudentDomain: assessment.klicker.stg.df-app.ch
+  appControlDomain: control.klicker.stg.df-app.ch
+  cookieDomain: .klicker.stg.df-app.ch
+
+  ingress:
+    className: haproxy
+    annotations:
+      cert-manager.io/cluster-issuer: k8s-crt-mgr-iss-stg-apps-default
+    hosts:
+      - host: api.klicker.stg.df-app.ch
+        paths:
+          - path: /
+            pathType: ImplementationSpecific
+      - host: backend-sls.klicker.stg.df-app.ch
```

Comment:
for stg, just use api. also for the .prd. version of the backend. we just CNAME backend-sls.klicker.uzh.ch to api.klicker.prd.df-app.ch in the end, so we need to list that in the prod values to get a certificate